### PR TITLE
Mysql client installation in linux docker

### PR DESCRIPTION
**Pull request by rokmicrosoft**

### CODE REVIEW

File: ubuntu/14.04/standard/Dockerfile [hunk 1]
```diff
@@ -117,6 +117,11 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
     powershell \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Mysql
+RUN apt-get update \
+ && apt-get install mysql-client -y
```
Comment (thechrisjohnson) [inline ubuntu/14.04/standard/Dockerfile]: We need to be sure and add "&& rm -rf /var/lib/apt/lists/*" to this set of commands, like the others, to reduce the size of the image's individual layers.

Comment (stepro) [reply ubuntu/14.04/standard/Dockerfile]: Also, I've been trying to keep the various tool installations loosely ordered alphabetically (although I just noticed Open**J**DK and **G**o aren't following this). Feel free to commit as-is and when I sync and update the README.md, I'll fix the ordering of OpenJDK and Go as well as putting MySQL right before the .NET SDK. Thanks!

Comment (rokmicrosoft) [reply ubuntu/14.04/standard/Dockerfile]: @thechrisjohnson  added "&& rm -rf /var/lib/apt/lists/*" it .Also tested . Please review and merge it .

**Comment by stepro** [comment]
The images have been updated.

**Comment by thechrisjohnson** [comment]
@stepro Did we update the ubuntu-16.04-docker-17.03.0-ce-standard image as well? I'm not seeing any changes there when I pull down the latest version.

**Comment by stepro** [comment]
No, when we introduced support for docker 17.12-0.ce we decided to drop updates for docker 17.03.0-ce because it exploded the number of permutations (see #65). The images remain on Docker hub but haven't been updated since adding docker 17.12-0.ce.

**Pull request merged**